{
  "public_data_at_usn": "0001-01-01T00:00:00Z",
  "crd": "0001-01-01T00:00:00Z",
  "candidate": "CVE-2021-34552",
  "public_date": "2021-07-13T17:15:00Z",
  "references": [
    {
      "reference": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-34552"
    },
    {
      "reference": "https://pillow.readthedocs.io/en/stable/releasenotes/index.html"
    },
    {
      "reference": "https://pillow.readthedocs.io/en/stable/releasenotes/8.3.0.html#buffer-overflow"
    }
  ],
  "description": "Pillow through 8.2.0 and PIL (aka Python Imaging Library) through 1.1.7 allow an attacker to pass controlled parameters directly into a convert function to trigger a buffer overflow in Convert.c.",
  "ubuntu_description": "",
  "notes": [
    {
      "note": "mdeslaur\u003e limited to a DoS only because of FORTIFY_SOURCE"
    }
  ],
  "bugs": [],
  "priority": "low",
  "discovered_by": "",
  "assigned_to": "",
  "patches": [
    {
      "package_name": "pillow",
      "release_patches": [
        {
          "release_name": "focal",
          "status": "needed",
          "note": ""
        }
      ]
    },
    {
      "package_name": "pillow-python2",
      "release_patches": [
        {
          "release_name": "focal",
          "status": "needs-triage",
          "note": ""
        }
      ]
    },
    {
      "package_name": "python-imaging",
      "release_patches": [
        {
          "release_name": "focal",
          "status": "DNE",
          "note": ""
        }
      ]
    }
  ],
  "upstreams": [
    {
      "package_name": "pillow",
      "upstream_links": [
        {
          "link": "https://github.com/python-pillow/Pillow/commit/31c473898c29d1b7cb6555ce67d9503a4906b83f"
        },
        {
          "link": "https://github.com/python-pillow/Pillow/commit/518ee3722a99d7f7d890db82a20bd81c1c0327fb"
        }
      ]
    }
  ]
}
